src/DocumentFormat.OpenXml/Schema/Presentation/CommentPropertiesExtension.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+
+namespace DocumentFormat.OpenXml.Office2021.PowerPoint.Comment
+{
+    /// <summary>
+    /// <para>Data for the Windows platform..</para>
+    /// <para>This class is available in Office 2021 and above.</para>
+    /// <para>When the object is serialized out as xml, it's qualified name is p:ext.</para>
+    /// </summary>
+    /// <remark>
+    /// <para>The following table lists the possible child types:</para>
+    /// <list type="bullet">
+    ///   <item><description><see cref="DocumentFormat.OpenXml.Office.PowerPoint.Y2022.M03.Main.Reactions" /> <c>&lt;p223:reactions></c></description></item>
+    ///   <item><description><see cref="DocumentFormat.OpenXml.Office.PowerPoint.Y2022.M08.Main.TaskDetails" /> <c>&lt;p228:taskDetails></c></description></item>
+    /// </list>
+    /// </remark>
+    [Obsolete("Unused type, prefer DocumentFormat.OpenXml.Presentation.CommentPropertiesExtension", true)]
```

Comment:
can you add `[EditorBrowsable(EditorBrowsableState.Never)]` as well?

Reply:
yup!